packages/snjs/lib/migrations/2_7_23.ts

```diff
@@ -0,0 +1,31 @@
+import { ApplicationStage } from '../stages';
+import { Migration } from '@Lib/migrations/migration';
+import { ContentType } from '@Lib/models';
+import { SettingName } from '@standardnotes/settings';
+
+export class Migration2_7_23 extends Migration {
+  static version(): string {
+    return '2.7.23';
+  }
+
+  protected registerStageHandlers(): void {
```

Comment:
Sign in event is only on explicit sign in. And actually you don’t have to be signed in to have a repo. This should be on full sync completed instead.

Reply:
But we need a user uuid to be able to update the user setting

Reply:
Ah ok, I’m that case on full sync completed and if signed in.